src/util.js

```diff
@@ -5,5 +5,5 @@ export function parseDateTime(datetime) {
   let [hour, minute, second] = time.split(":");
   let [day, month, year] = date.split("-");
 
-  return new Date(year, parseInt(month)-1, day, hour, minute, second);
+  return new Date(Date.UTC(year, parseInt(month)-1, day, hour, minute, second));
```

Comment:
That...makes more sense...I wasn't paying attention to what my server set to (which is UTC), but browser is set to Central US time.
I get this error on the chart: Invalid data at row #28: start(Mon Mar 16 2015 17:23:45 GMT-0500 (Central Daylight Time)) > end(Mon Mar 16 2015 12:24:12 GMT-0500 (Central Daylight Time)).
Is there a reason for choosing that string format for dates? Could it instead be ISO formatted and then the JS could just call Date.parse()? as an easier fix than trying to convert all the dates to/from UTC